src/test/scala/lms/thirdparty/test_cuda.scala

```diff
@@ -242,5 +242,65 @@ class CudaTest extends TutorialFunSuite {
     System.out.println(indent(driver.code))
   }
 
+  test("kernel_performance") {
+    // Based on example shown in:
+    // https://developer.nvidia.com/blog/how-implement-performance-metrics-cuda-cc/
+    val driver = new DslDriverCCuda[Int, Unit] {
+      
+      @virtualize
+      def snippet(arg: Rep[Int]) = {
+        
+        val saxpy = cudaGlobalFun[Int, Float, Array[Float], Array[Float], Unit]((n, a, x, y) => {
+          val i = blockIdxX * blockDimX * threadIdxX
+          if (i < n) {
+            y(i) = a * x(i) + y(i)
+          }
+        })
```

Comment:
Thanks! can we put this as part of the `cuda.scala` code so that it can be used by any other tests?
it is sort of like a library of cuda kernel functions that we are building